src/main/resources/liquibase/crop_changelog/15_4_1.xml

```diff
@@ -13,7 +13,7 @@
 		</preConditions>
 		<comment> REMOVE Study Detail AS DEFAULT TYPE FOR environment-LEVEL/plot-LEVEL variables. IBP-3889
 		</comment>
-		<sql dbms="mysql" splitStatements="TRUE">
+		<sql dbms="mysql" splitStatements="true">
```

Comment:
@mchabito - please also note the changes in this commit. I made "true" and "and" smallcaps.